mappings/net/minecraft/client/gui/ScreenChild.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cws net/minecraft/client/gui/InputListener
+CLASS cws net/minecraft/client/gui/ScreenChild
```

Comment:
It's referenced as `child` in a field but I doubt it's the Mojang name. I imagine this'd be something like `ScreenElement`/`ScreenComponent`

Reply:
Screen components already exist...really it's just a clickable object. Any suggestions?

Reply:
Huh. The current ScreenComponent name seems wrong -- it is just a more specific implementation of `ScreenChild`.

Reply:
As it is, it doesn't hold much 'component'-like behaviour, despite implementing `ScreenChild`.

Reply:
oh. Hmm I have an idea for a more thorough PR but it will be controversial I think.